### ISSUE TITLE
Allow Ability to Resurface Bounties to the Top When Expiration Date is Extended

### ISSUE THREAD

**Issue by PixelantDesign** [gitcoinco/web#4744]
### User Story

As a Gitcoin Admin, I'd like the ability for issues I've re-marketed to surface to the top of the issue explorer as a newly opened issue.

As a Contributor, I'd like to see re-marketed issues at the top of the issue explorer so I'll know that it is open and available to be worked on.

### What 
When an issue is remarketed, it should surface to the top of the issue explorer and show up as a new issue in notifiers-gitcoin on slack.

### Why
This will help us move issues through the platform.

### Definition of Done
- [ ] Build a feature that allows for surfacing to the top of the issue explorer when expiration has been extended.
- [ ] Post a video or screenshots of the feature built
- [ ] This issue passes all of the tests and revisions from the Gitcoin Core engineering team.

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 125.0 DAI (125.0 USD @ $1.0/DAI)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/4744/3210).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/4744/3210).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $137,130.09 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by PixelantDesign** [comment]
Reserved for @rafalkowalski

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 12 months from now.
Please review their action plans below:


**1) [rafalkowalski](https://gitcoin.co/profile/rafalkowalski) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/gitcoinco/web/4744/3210?mutate_worker_action=approve&worker=rafalkowalski) | [reject worker](https://gitcoin.co/issue/gitcoinco/web/4744/3210?mutate_worker_action=reject&worker=rafalkowalski))_.**

I would like to work on resurfacing bounties

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/4744/3210).

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 12 months from now.
Please review their action plans below:


**1) [rafalkowalski](https://gitcoin.co/profile/rafalkowalski) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/gitcoinco/web/4744/3210?mutate_worker_action=approve&worker=rafalkowalski) | [reject worker](https://gitcoin.co/issue/gitcoinco/web/4744/3210?mutate_worker_action=reject&worker=rafalkowalski))_.**

I would like to work on resurfacing bounties

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/4744/3210).

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by gitcoinbot** [comment]
@rafalkowalski Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by owocki** [comment]
@rafalkowalski i'm removing you from the bounty

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 11 months, 2 weeks from now.
Please review their action plans below:


**1) [vince0656](https://gitcoin.co/profile/vince0656) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/gitcoinco/web/4744/3210?mutate_worker_action=approve&worker=vince0656) | [reject worker](https://gitcoin.co/issue/gitcoinco/web/4744/3210?mutate_worker_action=reject&worker=vince0656))_.**

Hi there,

I am currently working on, but wrapping up, the re-market resurfacing functionality (GitHub issue 4745) and I would love to work on this next please. 

I would be extending the re-surfacing functionality in that ticket to affect issues that have their expiration extended.

Any questions, let me know.

Kind Regards,
Vincent

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/4744/3210).

**Comment by vince0656** [comment]
Hi @PixelantDesign,

I'm making good progress on this but I'm also waiting for #4876 to be merged into master so that I can open a PR as this work builds on top of the work done for #4745. 

I do have a question about this work. What happens if someone has already re-marketed their issue twice (the limit of how many times you can re-market)? I'm assuming that when you extend the issue expiration, you still want it to re-surface with a re-market but then what prevents abuse of extend expiration as a way of re-marketing an issue and therefore circumventing the re-market limit? A solution that I've come up with is that if the issue has already been re-marketed twice, when extending expiration, we can re-market one more time but the next time a bounty is extended, it won't be re-marketed and therefore won't surface to the top of the issue explorer. This would therefore leave a total re-marketing limit of 3 re-markets per issue.

Hope that makes sense. If not, please let me know and I'll explain further.

Just a side note that I do think that the re-market limit lends itself well into a paid model where once you've hit the re-market limit, you can then pay for more re-markets which surface the issue to the top of the issue explorer :) 

Thanks,
Vincent

**Comment by gitcoinbot** [comment]
@vince0656 Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by vince0656** [comment]
Hi there,

I am still working on this but waiting on @PixelantDesign for more input

Thanks,
Vincent

**Comment by vince0656** [comment]
Hi there,

I am still working on this but waiting on @PixelantDesign for more input

Thanks,
Vincent

**Comment by PixelantDesign** [comment]
Hi @vince0656 

>What happens if someone has already re-marketed their issue twice (the limit of how many times you can re-market)? I'm assuming that when you extend the issue expiration, you still want it to re-surface with a re-market but then what prevents abuse of extend expiration as a way of re-marketing an issue and therefore circumventing the re-market limit? A solution that I've come up with is that if the issue has already been re-marketed twice, when extending expiration, we can re-market one more time but the next time a bounty is extended, it won't be re-marketed and therefore won't surface to the top of the issue explorer. This would therefore leave a total re-marketing limit of 3 re-markets per issue.

I think I answered this on the PR?

We want to limit it to twice.
They'll get messages that say this is the last time you can remarket.
cc @thelostone-mc

**Comment by vince0656** [comment]
Ok thanks for clarifying @PixelantDesign! Therefore, regardless of whether someone directly re-markets or whether a re-market is triggered by expiration extension, the total allowed is 2 re-markets where appropriate messaging is given to the user to warn them as per #4876. Cc @thelostone-mc

**Comment by gitcoinbot** [comment]
@vince0656 Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/4744/3210?snooze=100)

**Comment by vince0656** [comment]
Hi there,

I’m still working on this issue. Shouldn’t be much longer

Thanks,
Vincent

**Comment by vince0656** [comment]
Hi there,

I’m still working on this issue. 

Thanks,
Vincent

**Comment by vince0656** [comment]
Hi there,

I’m still working on this issue.

Thanks,
Vincent

**Comment by vince0656** [comment]
Hi there,

I’m still working on this issue.

Thanks,
Vincent

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. **Submitted** 4. Done 

<hr>

__Work for 125.0 DAI (125.0 USD @ $1.0/DAI) has been submitted by__:

 1. [@vince0656](https://gitcoin.co/profile/vince0656)



@ceresstation please take a look at the submitted work:
* [PR](https://github.com/gitcoinco/web/pull/5012) by @vince0656

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/4744/3210)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/4744/3210).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $135,331.30 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by thelostone-mc** [comment]
We are good for payout ^_^ thanks man @vince0656  🙌

**Comment by thelostone-mc** [comment]
@ceresstation / @PixelantDesign  could we pay this out ^_^

**Comment by vince0656** [comment]
Hey @ceresstation / @PixelantDesign / @thelostone-mc. Sorry to bug, but any idea on when this will be paid out? Thanks :)

**Comment by vince0656** [comment]
Hey @ceresstation / @PixelantDesign / @thelostone-mc. Sorry to bug, but any idea on when this will be paid out? PR that was merged into master: https://github.com/gitcoinco/web/pull/5012 Thanks :)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__The funding of 125.0 DAI (125.0 USD @ $1.0/DAI)  attached to this issue has been approved & issued to @vince0656.__  

 
* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/4744/3210)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $152,055.54 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
